curriculum/get-challenges.js

```diff
@@ -217,48 +221,24 @@ async function buildChallenges({ path: filePath }, curriculum, lang) {
   }
   const { meta } = challengeBlock;
   const isCert = path.extname(filePath) === '.yml';
+  const englishPath = path.resolve(__dirname, CHALLENGES_DIR, lang, filePath);
```

Comment:
Is this an accurate name? Is `lang` always going to be `english` here?

Reply:
:man_facepalming:

Reply:
Haha it's okay, I speak "facepalm" - goodness knows I have enough of those moments myself